R/namespace_linter.R

```diff
@@ -85,9 +85,14 @@ namespace_linter <- function(check_exports = TRUE, check_nonexports = TRUE) {
     # run here, not in the factory, to allow for run- vs. "compile"-time differences in package structure
     namespaces <- lapply(packages, function(package) tryCatch(getNamespace(package), error = identity))
     failed_namespace <- vapply(namespaces, inherits, "condition", FUN.VALUE = logical(1L))
+    # nocov start
     if (any(failed_namespace)) {
-      stop("Failed to retrieve namespaces for one or more of the packages. Please report this issue.") # nocov
+      stop(
+        "Failed to retrieve namespaces for one or more of the packages. ",
+        "Please report an issue at https://github.com/r-lib/lintr/issues."
+      )
```

Comment:
Similar to what we do here:

https://github.com/r-lib/lintr/blob/becdbac98bc32c4719539e1d3d5263025b63c738/R/object_usage_linter.R#L295-L298